helix-term/src/ui/mod.rs

```diff
@@ -93,13 +93,14 @@ pub fn regex_prompt(
     )
 }
 
-pub fn file_picker(root: PathBuf) -> FilePicker<PathBuf> {
+pub fn file_picker(root: PathBuf, config: &helix_view::editor::Config) -> FilePicker<PathBuf> {
     use ignore::{types::TypesBuilder, WalkBuilder};
     use std::time;
 
     // We want to exclude files that the editor can't handle yet
     let mut type_builder = TypesBuilder::new();
     let mut walk_builder = WalkBuilder::new(&root);
```

Comment:
There's another `WalkBuilder` in commands.rs used for global search, we should pass this option there too.

Reply:
In commit [339325f](https://github.com/helix-editor/helix/pull/988/commits/339325f58b4140793c4c97783d042e51595609ad) I added a variable to allow for passing into `.git_ignore()` argument without fiddling with explicit lifetimes, and then inserted this call right after `WalkBuilder::new(...)` is called.

The remaining changes were all due to formatting, using Helix with rust-analyser.